packages/node-integration-tests/suites/send-event/capture-message/scenario.ts

```diff
@@ -0,0 +1,8 @@
+import * as Sentry from '@sentry/node';
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+});
+
+Sentry.captureMessage('Message');
```

Comment:
let's call this `suites/public-api/capture-message/...` to better match the playwright tests?